### PR TITLE
Bump version and update mainnet config

### LINKED ISSUE #880: Bump Lisk SDK version to 6.0.0
### Description

Update Lisk SDK version to 6.0.0 once published in public registry

### Acceptance Criteria

- Should be able to run node locally as mainnet

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #880 

### How was it solved?

- Bump sdk and deps version
- Update mainnet config

### How was it tested?

- Run core with mainnet flag

### CODE REVIEW

File: config/mainnet/config.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		"blockTime": 10,
 		"chainID": "00000000",
 		"maxTransactionsSize": 15360,
-		"minimumCertifyHeight": 1
+		"minimumCertifyHeight": 23461753
```
Comment (nagdahimanshu) [inline config/mainnet/config.json:20]: As migration height is already decided, we can update `mainnet` config to include backup height as well.

Comment (nagdahimanshu) [reply config/mainnet/config.json:20]: Sorry for the confusion. Its needed for Lisk-core v3

Comment (ManuGowda) [reply config/mainnet/config.json:20]: ah i thought you added that as reminder for migrator. we need to update migrator with config?

**Review by priojeetpriyom — APPROVED** [review]

**Review by Nazgolze — APPROVED** [review]

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**